### PR TITLE
[cli/update] Add Update functionality for secrets

### PR DESCRIPTION
**Pull request by tenczar**

### CODE REVIEW

File: pkg/dispatchcli/cmd/update.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"github.com/spf13/cobra"
+	"github.com/vmware/dispatch/pkg/dispatchcli/i18n"
+)
+
+var (
+	updateLong = i18n.T(`Create a resource. See subcommands for resources that can be created.`)
+
+	// TODO: Add examples
+	updateExample = i18n.T(``)
+	//file          = i18n.T(``)
```
Comment (berndtj) [inline pkg/dispatchcli/cmd/update.go]: delete this

File: pkg/dispatchcli/cmd/update.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+			if file == "" {
+				runHelp(cmd, args)
+				return
+			}
+
+			updateMap := map[string]modelAction{
+				"secret": CallUpdateSecret,
+			}
+
+			err := importFile(out, errOut, cmd, args, updateMap)
```
Comment (berndtj) [inline pkg/dispatchcli/cmd/update.go]: Does this actually work?  It doesn't seem as though you've added a file argument to the update command.

File: pkg/secret-store/service/k8s_secrets_service.go [hunk 3]
```diff
@@ -131,20 +132,28 @@ func (secretsService *K8sSecretsService) DeleteSecret(name string) error {
 }
 
 func (secretsService *K8sSecretsService) UpdateSecret(secret models.Secret) (*models.Secret, error) {
-	var entities []*secretstore.SecretEntity
+	entity := secretstore.SecretEntity{}
+	name := *secret.Name
 
+<<<<<<< HEAD
```
Comment (berndtj) [inline pkg/secret-store/service/k8s_secrets_service.go]: Fix the merge

Comment (tenczar) [reply pkg/secret-store/service/k8s_secrets_service.go]: oh... It looks like my IDE does not automatically save after I tell it to take one change over another... I can fix this quickly.

File: pkg/dispatchcli/cmd/update.go (hunk 2 above)
Comment (tenczar) [reply pkg/dispatchcli/cmd/update.go]: I admit that I took this from the code that sets up the create command. Now that I look at this, I'm just as confused as you are. It compiles and runs properly, but I can't tell where that `file` variable comes from.

Comment (tenczar) [reply pkg/dispatchcli/cmd/update.go]: Actually I have an idea... I believe file is declared as a package scoped variable in the create.go file. In that file it is declared as `file = i18n.T(\`\`)`

**Pull request merged**